### PR TITLE
adding clarification

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2260

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/upn-changes.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ A user's OneDrive URL is based on their UPN:
 
 (where user1_contoso_com corresponds with user1@contoso.com)
 
+> [!NOTE]
+> If the user's UPN contains any underscore, it will be present in the resultant OneDrive URL.
```
Comment (JohanFreelancer9) [inline OneDrive/upn-changes.md:65]:
```suggestion
> If the user's UPN contains an underscore, it will be present in the resultant OneDrive URL.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**